File: app/src/main/java/com/so/luotk/models/youtubeEx/youtube/playerResponse/StreamSelectionConfig.java
package com.so.luotk.models.youtubeEx.youtube.playerResponse;

import java.io.Serializable;

public class StreamSelectionConfig implements Serializable {
    private String maxBitrate;

    public String getMaxBitrate() {
        return maxBitrate;
    }

    public void setMaxBitrate(String maxBitrate) {
        this.maxBitrate = maxBitrate;
    }

    @Override
    public String toString() {
        return
                "StreamSelectionConfig{" +
                        "maxBitrate = '" + maxBitrate + '\'' +
                        "}";
    }
}

File: app/src/main/java/com/so/luotk/models/youtubeEx/newModels/StreamSelectionConfig.kt
package com.so.luotk.models.youtubeEx.newModels

import com.google.gson.annotations.SerializedName

class StreamSelectionConfig {
    @SerializedName("maxBitrate")
    var maxBitrate: String? = null
    override fun toString(): String {
        return "StreamSelectionConfig{" +
                "maxBitrate = '" + maxBitrate + '\'' +
                "}"
    }
}